bff/hosts/Hosts.IdentityServer/Config.cs

```diff
@@ -118,4 +119,10 @@ private static Client BuildClient(string clientId, Action<Client> postConfigure,
         AccessTokenLifetime = 15 // Force refresh
     };
 
+    private static Uri CombineUriPath(Uri baseUri, string relativePath)
+    {
+        var basePath = baseUri.AbsolutePath.TrimEnd('/');
+        var relPath = relativePath.TrimStart('/');
+        return new Uri(baseUri, $"{basePath}/{relPath}");
```

Comment:
Smell here with the string manipulation. Should be able to do this robustly through the URI class.

Reply:
agreed.. had an issue with relative / absolute pathing but didn't think it through entirely. Working better now in latest commit.